src/test.jest.testRunner.ts

```diff
@@ -164,8 +166,26 @@ async function askRunner(
     title: 'compiles',
   });
 
+  // Adding Jest functions to the environment, so that we can run Jest expect():toBe() in .ask files.
+  // See a sample in src/askscript/__tests__/00-documentation-examples/documentation01-complete_example.test.ask
+  const testResources = {
+    expect: resource({
+      type: any,
+      async resolver(actual: any): Promise<e.Matchers<any>> {
+        return e(actual);
+      },
+    }),
+
+    toBe: resource({
+      type: any,
+      async resolver(matcher: e.Matchers<any>, actual: any): Promise<void> {
+        return matcher.toBe(actual);
+      },
+    }),
+  };
```

Comment:
Can we add more? The are all going to be implemented in a similar way - so this shouldn't require to much work without code repetition

Reply:
Of course we could.  Let's discuss which resources to add: https://github.com/xFAANG/askql/issues/290

Reply:
Sure, we can do it as a follow up